src/dmd/semantic2.d

```diff
@@ -99,7 +99,10 @@ private extern(C++) final class Semantic2Visitor : Visitor
 
         import dmd.staticcond;
         bool errors;
-        bool result = evalStaticCondition(sc, sa.exp, sa.exp, errors);
+        Expression msg;
+        bool result = evalStaticCondition(sc, sa.exp, sa.exp, errors,null, &msg);
```

Comment:
Could this code detetct and expand `TupleExp`  here instead of specialising `evalStaticCondition`?

Reply:
No, It will need to be used by `evaluateConstraint` as well